### PR TITLE
docstring cleanup for scenarios

### PR DESCRIPTION
**Pull request by AboudyKreidieh**

### CODE REVIEW

File: flow/scenarios/__init__.py [hunk 1]
```diff
@@ -25,20 +27,12 @@
...
-    "CircleGenerator", "MergeGenerator", "NetFileGenerator",
-    "TwoLoopOneMergingGenerator"
-]
-
-# custom scenarios
-__all__ += [
-    "BayBridgeScenario", "BayBridgeTollScenario", "BottleneckScenario",
-    "Figure8Scenario", "SimpleGridScenario", "HighwayScenario", "LoopScenario",
-    "MergeScenario", "NetFileScenario", "TwoLoopsOneMergingScenario"
+__all__ = [
```
Comment (eugenevinitsky) [inline flow/scenarios/__init__.py]: Why do this? This seems less clean.

File: flow/scenarios/bay_bridge_toll/gen.py [hunk 2]
```diff
@@ -1,13 +1,14 @@
+"""Contains the Bay Bridge toll generator class."""
+
 from flow.scenarios.netfile.gen import NetFileGenerator
 
 
 class BayBridgeTollGenerator(NetFileGenerator):
-    """
-    Bay Bridge bottleneck generator.
-    """
+    """Bay Bridge bottleneck generator."""
```
Comment (eugenevinitsky) [inline flow/scenarios/bay_bridge_toll/gen.py]: seems misnamed given the above information?

File: flow/scenarios/bottleneck/gen.py [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+
 from flow.core.generator import Generator
 import numpy as np
 
 
 class BottleneckGenerator(Generator):
-    """
-    Generator class for simulating the Bay Bridge toll. No parameters needed
-    from net_params (the network is not parametrized)
+    """Generator class for simulating the Bay Bridge toll.
```
Comment (eugenevinitsky) [inline flow/scenarios/bottleneck/gen.py]: Seems misnamed given the above naming scheme

File: flow/scenarios/loop/gen.py [hunk 4]
```diff
@@ -1,27 +1,23 @@
...
 from flow.core.generator import Generator
 
 from numpy import pi, sin, cos, linspace
 
 
 class CircleGenerator(Generator):
-    """
-    Generator for loop circle used in MIT traffic simulation.
-    """
+    """Generator for loop circle used in MIT traffic simulation."""
```
Comment (eugenevinitsky) [inline flow/scenarios/loop/gen.py]: Remove MIT?

File: flow/scenarios/loop_merge/gen.py [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
 from numpy import pi, sin, cos, linspace
 
 
 class TwoLoopOneMergingGenerator(Generator):
-    """
-    Generator for a two-loop network in which both loops merge into a common
-    lane.
+    """Generator for a two loop merge network.
+
+    This network conssits of two loops that both merge into a common lane.
```
Comment (eugenevinitsky) [inline flow/scenarios/loop_merge/gen.py]: Consists

File: flow/scenarios/__init__.py (hunk 1 above)
Comment (AboudyKreidieh) [reply flow/scenarios/__init__.py]: Docstring conventions don't like it when we do this:
```
WARNING: Error in file flow/scenarios/__init__.py: Could not evaluate contents of __all__. 
That means pydocstyle cannot decide which definitions are
public. Variable __all__ should be present at most once in
each file, in form
```

Comment (eugenevinitsky) [reply flow/scenarios/__init__.py]: kk, can't do much about that

**Comment by AboudyKreidieh** [comment]
Thanks for the thorough review.

**Pull request merged**